### ISSUE TITLE
Docker Image: User `postgres' has no password assigned

### ISSUE THREAD

**Issue by ilfrich** [minus34/gnaf-loader#76]
Hi again, trying out the Nov22 docker image, I ran into another issue.

TLDR; after running the database and trying to connect (from outside the container), I got:

```
2023-03-20 01:45:44.820 UTC [81] postgres@postgres FATAL:  password authentication failed for user "postgres"
2023-03-20 01:45:44.820 UTC [81] postgres@postgres DETAIL:  User "postgres" has no password assigned.
Connection matched pg_hba.conf line 105: "host all  all    0.0.0.0/0  md5"
```

**Steps**

- Downloaded the docker image
- Run `docker run --publish=5433:5432 --name=gnafloader minus34/gnafloader:202211-gda2020`
- Try to connect from outside via Beekeper Studio or `psycopg2`
- Authentication failure with the error message mentioned above `User "postgres" has no password assigned.`
- Connected to the docker container `docker exec -it gnafloader /bin/bash` and ran `psql --port=5433` and execute `ALTER USER postgres PASSWORD "password";`
- Now the connection is established successful

**Potential Issue**

It's weird, because setting the password is in the `Dockerfile`. My best guess would be that maybe the `pg_restore` command overwrites the previously set password and sets it back to nothing, if your dump contained no password for the `postgres` user.

**Comment by minus34** [comment]
This is unexplained. Yes, it should be set during container build. Also, pg_restore doesn't override the database password.

Please check this using the 202302 container i just re-released as linux/amd64. I've tested with pgAdmin 4 and it connects as expected.

**Comment by ilfrich** [comment]
Just downloaded the latest `minus34/gnafloader:latest-gda2020` and it worked straight out of the box :+1: 
Must have been an issue with the Nov22 image :man_shrugging:
